README.md

```diff
@@ -1,16 +1,52 @@
 <img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
 
-# openmrs-addOnManager
+# openmrs Add-on Manager
 
 This repository contains the openmrs-addOnManager OpenMRS Open Web App.
 
-> Add a description of what your app does here.
+> The openmrs-addOnManager is a tool used for uploading, listing, and deleting existing OWAs and modules.
```

Comment:
Why not this? The OpenMRS Add-Ons Manager is a tool ....

Reply:
Nice, thanks @dkayiwa